e2e/test/scenarios/metrics/metrics-editing.cy.spec.js

```diff
@@ -407,8 +407,9 @@ describe("scenarios > metrics > editing", () => {
       H.enterCustomColumnDetails({
         formula: `[${ORDERS_SCALAR_METRIC.name}] / 2`,
         name: "",
+        blur: true,
       });
-      H.popover().button("Update").click();
+      H.popover().button("Update").should("not.be.disabled").click();
```

Comment:
this should not be needed as `.click()` checks for this implicitly, [see docs](https://docs.cypress.io/app/core-concepts/interacting-with-elements#Actionability)